File: programmers/java/basic/181868.java
// 공백으로 구분하기 2

import java.util.Arrays;

class Solution {
    public String[] solution(String my_string) {
        return Arrays.stream(my_string.split(" "))
            .filter(str -> !str.equals(""))
            .toArray(String[]::new);
    }
}

File: programmers/js/basic/181868.js
// 공백으로 구분하기 2
function solution(my_string) {
  return my_string.split(" ").filter((str) => str !== "");
}

// 다른 풀이
function solution(my_string) {
  return my_string.trim().split(/ +/);
}
